src/main/java/ch/njol/skript/expressions/ExprDrops.java

```diff
@@ -42,7 +34,15 @@
 import ch.njol.skript.util.Experience;
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
-import ch.njol.util.coll.iterator.IteratorIterable;
+import org.bukkit.event.Event;
+import org.bukkit.event.entity.EntityDeathEvent;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.ArrayList;
```

Comment:
Java imports come first, why is your IDE formatting them to be last?

Reply:
We have import code standards to avoid frequent import changes in commits.

Reply:
They do??? This is a default intellij setting from what I remember.

Reply:
Looking over skript, more and more recently java imports have been second with older code being on top.

Reply:
It doesn't happen with others, unless they changed it randomly. All of Skript follows Java imports first.

Reply:
That being both Section and Structure classes being the most recent larger apis added.

Reply:
> We have import code standards to avoid frequent import changes in commits.

Nowhere in the code conventions file is there anything mentioning import order. Can you please point me to where it clearly defines an import order standard to follow, in case I missed it?

Reply:
This is him referring to the import convention pr

Reply:
Pull request

Reply:
Well then it's not a requirement until it's merged, let's not get too ahead of ourselves, as it is subject to change. This review may be discarded.

Reply:
As tud said, these are not required conventions, nor do I think that imports are worth policing at all. IDEs collapse them for a reason.

Reply:
Can these changes be omitted from your commits then?

Reply:
Is the whole point of sovde's message just skipped across for you? This is a minute detail that means nothing, as I said on the PR for your import rules, we should be following Intellij/Jetbrains as this is the majority of users.

Reply:
> Can these changes be omitted from your commits then?

I suppose, but I think it's rather frivolous to worry about import order.

Reply:
If this is the hill you want to die on, I'll change them, but otherwise I'll leave them be. Let me know if you really want them changed and I'll do so.

Reply:
Yes. We cannot be changing the order every commit between developers.